app/views/playlists/_edit_form.html.erb

```diff
@@ -82,15 +82,15 @@ Unless required by applicable law or agreed to in writing, software distributed
           <div class="row">
             <div class="col-sm-2">
               <span class="fa fa-arrows-v float-left dd-handle"></span>
-              <%= text_field_tag "playlist[items_attributes[#{index}[position]]]", i.position, class: 'form-control position-input', form: 'playlist_sort_form' %>
+              <%= text_field_tag "playlist[items_attributes[#{index}[position]]]", i.position, class: 'form-control position-input', form: 'playlist_sort_form', 'aria-label': 'Playlist item position' %>
               <%= hidden_field_tag "playlist[items_attributes[#{index}[id]]]", i.id, form: 'playlist_sort_form' %>
             </div>
             <% if can? :read, i.clip.master_file %>
               <div class="col-sm-8 title" style="margin-top:5px">
                 <%= link_to i.clip.title, i.clip.mediafragment_uri, id: "playlist_item_title_label_#{i.id}" %>
               </div>
               <div class="col-sm-2 text-right" style="margin-top:5px">
-                <button id="playlist_item_edit_button" class="btn btn-sm fa fa-edit" <%= i.clip.master_file.nil? ? 'disabled' : '' %>
+                <button id="playlist_item_edit_button" class="btn btn-sm fa fa-edit" aria-label="Edit playlist item" <%= i.clip.master_file.nil? ? 'disabled' : '' %>
```

Comment:
I think it would be helpful to add the playlist item position (index + 1) to the `aria-label` for better clarity when using them with a screen reader. For an example: when navigating the page with Web rotor in MacOS' VoiceOver.